api/types/saml_idp_service_provider.go

```diff
@@ -129,6 +129,10 @@ type SAMLIdPServiceProvider interface {
 	GetACSURL() string
 	// SetACSURL sets the ACS URL.
 	SetACSURL(string)
+	// GetPreset returns the Preset.
+	GetPreset() string
+	// SetPreset sets the Preset.
+	SetPreset(string)
```

Comment:
This seems to be not used and only source of preset is yaml/json marshaling.

Reply:
Good point, removed in https://github.com/gravitational/teleport/pull/39385/commits/a7788f0f7f6e9890ff705ce6c2f73f11d8a55b2d